build/tasks/clang-format.js

```diff
@@ -8,7 +8,11 @@ var gulp = require('gulp');
 var format = require('gulp-clang-format');
 var clangFormat = require('clang-format');
 
-var tsSources = ['src/**/*.ts', '!src/sample-app/aot-compiled/**/*'];
+var tsSources = [
+    'src/app/**/*.ts',
+    '!src/ks-app/node_modules',
+    '!src/sample-app/aot-compiled/**/*'
+];
 
```

Comment:
Just curious, do we care about formatting the actual code inside `ks-app`? If so, we should add that here. If not, then we can remove the second and third globs as they won't be included anyways if we just keep the first one (`src/app/**/*.ts`).

Reply:
I included it with the thought that we should format it the same as the rest of the codebase. But, I'm not tied to that if we don't want to include it in the formatter.

Reply:
I think it should be formatted. I think I read the first glob as one directory up. I'll add it in.